README.md

```diff
@@ -39,11 +53,25 @@ def mystery2(n)
 end
 ```
 
-- What is mystery2(123)?
-- What is mystery2(9005)?
-- What is mystery2(-123)?
+- What is mystery2(123)?    6
+- What is mystery2(9005)?   14
+- What is mystery2(-123)?  -123
 - _Added Fun: How could we make `mystery2(-123)` work the way we might expect it to work instead of the way it does?_
 
+We could "ignore the negative sign" while we sum the digits and then replace it at the end. This could be done as follows:
+
+def mystery2B(n)
+  if n < 10 && n > -10
+    return n
+  elsif n >= 10
+    return (n%10) + mystery2(n/10)
+  else m = n.abs
+    a = (m%10) + mystery2(m/10)
+    return -a
+  end
+end
```

Comment:
This looks like it works. I also just posted an alternative solution that is perhaps more readable.